pycbc/filter/resample.py

```diff
@@ -178,6 +178,29 @@ def resample_to_delta_t(timeseries, delta_t, method='butterworth'):
 _highpass_func = {numpy.dtype('float32'): lal.HighPassREAL4TimeSeries,
                  numpy.dtype('float64'): lal.HighPassREAL8TimeSeries}
 
+def notch_fir(timeseries, f1, f2, order, beta=5.0):
+    """ notch filter the time series using an FIR filtered generated from 
+    the ideal response passed through a kaiser window (beta = 5.0)
+
+    Parameters
+    ----------
+    Time Series: TimeSeries
+        The time series to be low-passed.
+    f1: float
+        The start of the frequency suppression.
+    f2: float
+        The end of the frequency suppression.
+    order: int
+        Number of corrupted samples on each side of the time series
```

Comment:
Preferable to suggest some values for 'order' rather than leaving it entirely up to the user?

Reply:
I'm not sure about the advice to leave at the moment.

Reply:
Well, what criteria would you use to choose 'order'.  Presumably the desired 'sharpness' (frequency resolution) of the notch sides expressed as a number of samples.  So n ~ f_s/df where f_s is the sampling frequency.

Reply:
The definition as 'number of corrupted samples' is technically correct but it really means just the (one-sided) length of the filter in samples, yes?  So prefer `Extent of the filter on either side of zero`.  Then I would suggest `To achieve frequency resolution df at sampling frequency fs, order should be at least fs/df`